packages/ui/formAlerts/index.spec.jsx

```diff
@@ -0,0 +1,185 @@
+import { mount } from "enzyme";
```

Comment:
why a .spec file is a better choice here than a .test file? will we have both types of files?